contracts/cw-dao/src/contract.rs

```diff
@@ -352,6 +382,29 @@ pub fn execute_update_config(
         .add_attribute("sender", info.sender))
 }
 
+pub fn execute_update_cw20_token_list(
+    deps: DepsMut,
+    env: Env,
+    info: MessageInfo,
+    mut to_add: Vec<Addr>,
+    to_remove: Vec<Addr>,
+) -> Result<Response<Empty>, ContractError> {
+    // Only contract can call this method
+    if env.contract.address != info.sender {
+        return Err(ContractError::Unauthorized {});
+    }
+    TREASURY_TOKENS.update(deps.storage, |mut list| -> StdResult<_> {
```

Comment:
This will become much easier with the Map type. It is a set so we can add and remove without worrying about duplicates.